File: 연습문제/12954_x만큼_간격이_있는_n개의_숫자.java
// https://school.programmers.co.kr/learn/courses/30/lessons/12954

class Solution {
    public long[] solution(int x, int n) {
        long[] answer = new long[n];

        for(int i = 0; i < n; i++) {
            answer[i] = x + x * i;
        }

        return answer;
    }
}

File: 프로그래머스/연습문제/12954_x만큼_간격이_있는_n개의_숫자.cpp
// https://school.programmers.co.kr/learn/courses/30/lessons/12954

#include <string>
#include <vector>

using namespace std;

vector<long long> solution(int x, int n) {
    vector<long long> answer;

    for(int i = 0; i < n; i++) {
        answer.push_back(x + x * i);
    }

    return answer;
}